src/static/client/app/signup/signup.component.scss

```
@@ -0,0 +1,110 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body, html {
+    height: 100%;
+    font-family: Arial, sans-serif;
+  }
+  
+  .signup {
+    background: linear-gradient(to right, #1632aa, #B30D2C);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+
+    .button-container {
+      display: flex;
+      justify-content: space-between;
+    }
+    
+    #icon {
+      width: 150px;  // Adjust as needed to change the size of the logo.
+      height: auto;
+      display: block;
+      margin: 0 auto 20px auto; // This centers the image and adds space below it.
+    }
+  
+    .wrapper {
+      max-width: 400px;
+      background-color: #ffffff;
```

```suggestion
      background-color: #fff;
```
You dont need all the f's :)

---

done